crates/swc/src/config/mod.rs

```diff
@@ -466,7 +466,17 @@ impl Options {
                 swc_plugin_runner::cache::init_plugin_module_cache_once(&experimental.cache_root);
             }
 
-            crate::plugin::plugins(Some(plugin_resolver), experimental, plugin_context)
+            let comments = if let Some(comments) = comments {
```

Comment:
```rust
            let comments = if let Some(comments) = comments {
                Some(comments.clone())
            } else {
                None
            };
```

I think this is same as `let comments = comments.clone()`?

Reply:
No, since comments is `Option<&'a SingleThreaded..>`, `comments.clone()` gives same instead of `Option<SingleThreaded..>` But also realized it can be simplifed to `cloned()` instead. Updated.